pkg/azuredisk/azuredisk.go

```diff
@@ -20,22 +20,20 @@ import (
 	"context"
 	"fmt"
 	"reflect"
-	"regexp"
 	"strings"
 	"time"
-	"unicode"
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-12-01/compute"
 	"github.com/container-storage-interface/spec/lib/go/csi"
-	"github.com/pborman/uuid"
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
 	"k8s.io/klog/v2"
-	"k8s.io/kubernetes/pkg/volume/util"
 	"k8s.io/mount-utils"
 
+	azureconsts "sigs.k8s.io/azuredisk-csi-driver/pkg/azureconstants"
```

Comment:
Use `consts` for consistent aliasing - this is the only file to use `azureconsts`

Reply:
`consts` was already taken by `consts "sigs.k8s.io/cloud-provider-azure/pkg/consts"` but I guess I can rename that import instead.